pkg/codegen/pcl/binder_resource.go

```diff
@@ -424,7 +424,8 @@ func (b *binder) bindResourceBody(node *Resource) hcl.Diagnostics {
 			attrNames.Add(attr.Name)
 
 			if typ, ok := objectType.Properties[attr.Name]; ok {
-				if !typ.ConversionFrom(attr.Value.Type()).Exists() {
+				conversion := typ.ConversionFrom(attr.Value.Type())
+				if !conversion.Exists() {
```

Comment:
Why this change?

Reply:
Just to be able to put a break point at the location 😄